pkg/processor/build/builder.go

```diff
@@ -1363,16 +1363,52 @@ func (b *Builder) commandsToDirectives(commands []string) (map[string][]function
 	currentDirective := "preCopy"
 
 	// iterate over commands
-	for _, command := range commands {
+	for i := 0; i < len(commands); i++ {
+		command := commands[i]
+
 		if strings.TrimSpace(command) == "@nuclio.postCopy" {
 			currentDirective = "postCopy"
 			continue
 		}
 
+		// check if the last character is backslash. if so treat this and the next command as one multi-line command
+		// example: commands = []{"echo 1\", "2\", "3"} -> "RUN echo 1 2 3"
+		aggregatedCommand := ""
+		for {
+
+			if strings.TrimSpace(command) == "@nuclio.postCopy" {
+				currentDirective = "postCopy"
+				break
+
+			} else if len(command) != 0 && command[len(command)-1] == '\\' {
+
+				// remove whitespace
+				command = command[:len(command)-1]
+
+				// add command to the aggregated multi-line command
+				aggregatedCommand += command
+
+				if len(commands) > i + 1 {
+
+					// check if the next command is continuing the multi-line
+					i++
+					command = commands[i]
+
+				} else {
+					break
+				}
+
+			} else {
+
+				aggregatedCommand += command
+				break
+			}
+		}
+
 		// add to proper directive. support only RUN
 		directives[currentDirective] = append(directives[currentDirective], functionconfig.Directive{
 			Kind:  "RUN",
-			Value: command,
+			Value: aggregatedCommand,
```

Comment:
```suggestion
			Value: strings.Join(aggregatedCommands, " ")
```

Reply:
I saw that the standard is that it shouldn't be separated by space, so I decided to just concatenate the commands right after

Reply:
elaborate on the standard please? as I see it, you trim each command ended separated by '/' and concat it without any space so that
```
echo 1\
2\
3\
``` 

will turn out to be `echo 123` **rather than** `echo 1 2 3`